Makefile

```diff
@@ -1,16 +1,22 @@
-.PHONY: all image package dist clean
+.PHONY: all image package dist deploy clean
 
-all: package
+all: image dist
 
 image:
 	docker build --tag amazonlinux:nodejs .
 
-package: image
-	docker run --rm --volume ${PWD}/lambda:/build amazonlinux:nodejs npm install --production
+awsconfigure:
+	docker run -it --rm --volume ${PWD}/../.aws:/root/.aws amazonlinux:nodejs aws configure
```

Comment:
It might be best to pass along the AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_DEFAULT_REGION environment variables rather than using the file.